mne/epochs.py

```diff
@@ -2164,6 +2166,10 @@ def __init__(self, data, info, events, tmin=0, event_id=None,
         if len(info['ch_names']) != data.shape[1]:
             raise ValueError('Info and data must have same number of '
                              'channels.')
+        if events is None:
+            n_epochs = len(data)
+            events = np.c_[np.arange(n_epochs), np.zeros(n_epochs),
+                           np.zeros(n_epochs)]
```

Comment:
And also `np.zeros(n_epochs, int)`, `np.ones(n_epochs, int)` otherwise this result will be a float array